spec/bullet_spec.rb

```diff
@@ -38,4 +38,22 @@
       end
     end
   end
+
+  describe '#start?' do
+    context 'when bullet is disabled' do
+      around(:each) do |example|
+        begin
+          old_enabled = Bullet.enable?
+          Bullet.enable = false
+          example.run
+        ensure
+          Bullet.enable = old_enabled
+        end
+      end
+
+      it 'should not be started' do
+        expect(Bullet.start?).to be(false)
+      end
+    end
+  end
```

Comment:
there's no way to guarantee that the state after the test would be the same as the test before disabling Bullet, i.e. the other specs may nondeterministically fail.